src/pages/guides/flutter.mdx

```diff
@@ -0,0 +1,307 @@
+---
+title: Using Web3Auth's Flutter SDK
+image: "/docs/contents/guides/banners/flutter.png"
+description: Learn how to use Web3Auth Plug and Play Core SDK in your Flutter app
+type: guide
+tags: [mobile, flutter, whitelabel, custom authentication, dapp share]
+date: 13th Sept 2022
+author: Web3Auth Team
+order: 8
+---
+
+import SetupWeb3AuthDashboard from "@site/src/common/guides/_setup-web3auth-dashboard.mdx";
+import Web3AuthPrerequisites from "@site/src/common/guides/_web3auth-prerequisites.mdx";
+import AndroidResponse from "@site/src/common/sdk/android/_response.mdx";
+import SEO from "@site/src/components/SEO";
+
+<SEO
+  title="Using Web3Auth's Flutter SDK"
+  description="Learn how to use Web3Auth Plug and Play Core SDK in your Flutter application"
+  image="https://web3auth.io/docs/contents/guides/banners/android.png"
+  slug="/guides/flutter"
+/>
+
+This guide will cover the basics of how to use the Web3Auth Flutter SDK in your Flutter application.
+
+Repository: https://github.com/Web3Auth/examples/raw/main/flutter
+
+## Quick Start
+
+```bash
+git clone https://github.com/Web3Auth/examples.git
+cd flutter
+# Open in Android Studio or VS Code
+```
+
+## How it works?
+
+---
+
+When integrating Web3Auth Flutter SDK with Social Login the flow looks something like this:
+
+![Web3Auth Core - Social Login Flow](/images/implicit-flow.svg)
+
+- When a user logs in with `Google`, Google sends a JWT `id_token` to the app. This JWT token is sent to the Web3Auth SDK's login function.
+
+- Finally, on successful validation of the JWT token, Web3Auth SDK will generate a private key for the user, in a self custodial way, resulting in
+  easy onboarding for your user to the application.
+
+## Prerequisites
+
+---
+
+<Web3AuthPrerequisites />
+
+- Android API version 21 or newer.
+- `compileSdkVersion` 32 or higher
+
+## Setup
+
+---
+
+### Setup your Web3Auth Dashboard
+
+<SetupWeb3AuthDashboard />
+
+- Add `{YOUR_APP_PACKAGE_NAME}://auth` in the **Whitelist URL** field of the Web3Auth Dashboard.
+
+### Configure Deep Link
+
+Open your app's `AndroidManifest.xml` file and add the following deep link intent filter to your sign-in activity:
+
+```xml
+<intent-filter>
+  <action android:name="android.intent.action.VIEW" />
+
+  <category android:name="android.intent.category.DEFAULT" />
+  <category android:name="android.intent.category.BROWSABLE" />
+
+  <!-- Accept URIs: {YOUR_APP_PACKAGE_NAME}://* -->
+  <data android:scheme="{YOUR_APP_PACKAGE_NAME}" />
+</intent-filter>
+```
+
+## Using the Web3Auth SDK
+
+---
+
+To use the Web3Auth SDK, you need to add the dependency of the respective platform SDK of Web3Auth to your project. To know more about the available
+SDKs, please have a look at this [documentation page](/docs/sdk).
+
+For this guide here, we will be talking through the [Web3Auth Flutter SDK](/sdk/flutter).
+
+### Installation
```

Comment:
ios installation is missing

Reply:
Didn't check the iOS part.

Reply:
Not sure about the `pod install Web3Auth`. It doestn't work inside the flutter terminal.